_posts/2015-04-05-faq.md

```diff
@@ -203,7 +203,9 @@ Some bugs are more important than others.  Stability and usability issues are ve
 
 ## I'm not a developer, can I still contribute
 
-Don't worry if you are not a proficient developer -- there are many ways to help out.  You can become proficient in the installation and configuration of a BigBlueButton server.  Each month, there are many new users in [bigbluebutton-setup](https://groups.google.com/forum/#!forum/bigbluebutton-setup) that need help with setup of BigBlueButton, especially setup behind a [firewall](#can-i-provide-external-access-to-a-bigbluebutton-server-behind-my-firewall).    You can [contribute a language file](/dev/localization.html). You could point out any errors to this documentation.  Such assistance reduces the support load on us and gives us more time to work on improving BigBlueButton.
+Don't worry if you are not a proficient developer -- there are many ways to help out. Documentation is good way to start contributing you can start here [Documentation](https://github.com/bigbluebutton/bigbluebutton.github.io), You can also become proficient in the installation and configuration of a BigBlueButton server.  Each month, there are many new users in [bigbluebutton-setup](https://groups.google.com/forum/#!forum/bigbluebutton-setup) that need help with setup of BigBlueButton, especially setup behind a [firewall](#can-i-provide-external-access-to-a-bigbluebutton-server-behind-my-firewall).    You can [contribute a language file](/dev/localization.html). You could point out any errors to this documentation.  Such assistance reduces the support load on us and gives us more time to work on improving BigBlueButton.
```

Comment:
```suggestion
Don't worry if you are not a proficient developer -- there are many ways to help out. Documentation is an easy and helpful way to start contributing.  You can follow the instructions on getting started [here](https://github.com/bigbluebutton/bigbluebutton.github.io). You can also become proficient in the installation and configuration of a BigBlueButton server.  Each month, there are many new users in [bigbluebutton-setup](https://groups.google.com/forum/#!forum/bigbluebutton-setup) that need help with setup of BigBlueButton, especially setup behind a [firewall](#can-i-provide-external-access-to-a-bigbluebutton-server-behind-my-firewall).    You can [contribute a language file](/dev/localization.html). You could point out any errors to this documentation.  Such assistance reduces the support load on us and gives us more time to work on improving BigBlueButton.
```

Reply:
Yes sounds much better :-)

Reply:
@satyaakam if you agree to this change, please adjust your pull request by changing the file in your fork (satyaakam:FAQ-change) to automatically add that commit to this pull request.